### PR TITLE
Minor grammar fixes.

### PR DESCRIPTION
**Pull request by pq**
From Wm (@wmleler). 

(He doesn't have the repo setup just yet so putting out there for him.)

👍 

@LarkAscending @eseidel

### CODE REVIEW

File: packages/flutter/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Flutter
 =======
 
 Flutter is a new way to build high-performance, cross-platform mobile apps.
-Flutter is optimized for today's, and tomorrow's, mobile devices. We are
+Flutter is optimized for today's -- and tomorrow's -- mobile devices. We are
```
Comment (Hixie) [inline packages/flutter/README.md]: can we use a real em dash? :-)

Comment (Hixie) [inline packages/flutter/README.md]: here, you can copy this one: —

**Comment by Hixie** [comment]
LGTM
...if you use the real em dash.

File: packages/flutter/README.md (hunk 1 above)
Comment (pq) [inline packages/flutter/README.md]: @wmleler: looks like we can! :)

**Pull request merged**